### DISCUSSION TITLE
New install standalone - No alerts at all.

### DISCUSSION THREAD

**Discussion by jrm360seclab** [Security-Onion-Solutions/securityonion#11341] [2.4]
### Version

2.4.1

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

32

### Storage for /

1TB

### Storage for /nsm

721

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Good day all, I just installed a new standalone, actually I have tried this 4 times not but after the install I got some error stating I need to check the /root/sosetup.log.

Total states run:     89
Total run time:    6.221 s
Importing Plays - NOTE: this will continue after installation finishes and could take an hour or more. Rebooting while the import is in progress will delay playbook imports.
2023-09-17T13:18:15Z | INFO | Enabling checkin at boot
2023-09-17T13:18:28Z | INFO | Executing command: salt-call schedule.enable -linfo --local
local:
    ----------
    changes:
        ----------
        schedule:
            enabled
    comment:
        Enabled schedule on minion.
    result:
        True
2023-09-17T13:19:04Z | INFO | Verifying setup
WARNING: Errors detected during setup

I have no alerts at all. Can Please get some help on how to  get this working?

Thank you in advanced.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
> I got some error stating I need to check the /root/sosetup.log.

What is the output of the following?

```
sudo cat /root/errors.log
```

```
sudo grep ERROR /root/sosetup.log
```

> I have no alerts at all.

Have you tried following the Troubleshooting Alerts section of the documentation?
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

**Reply by jrm360seclab** [reply]
Thank you @dougburks , I completely, deleted the server. I created a new distribution install but now I am getting the 

Request failed with status code 500
See the Help section of the Security Onion documentation for additional troubleshooting guidance. 
The Hunt interface may contain additional log messages to further diagnose the issue. 

This is a new install.

**Reply by dougburks** [reply]
So you did a fresh OS installation on the same hardware using our ISO image?

Did you choose the standalone option again?

Were there any errors during Setup this time?

Have you checked the logs in /opt/so/log/ for additional clues?

**Reply by jrm360seclab** [reply]
I deleted the VM, created a new VM but on the same server host of course, but everything was built from scratch.
I used the same ISO that I downloaded from the site.
I didn't choose the Standalone, I did distributed with one Forward and one Search with the Master. No, I didn't get any error messages after the installation.  I have an instance of 2.4 running in one area, I am trying to set it up in a new environment and it hasn't been working. I will try to download a new iso, rebuild the VMs again and see if that this works.

**Reply by jrm360seclab** [reply]
This is from a Brand new install:

[root@so-mngr]# sudo cat /root/errors.log
### Cleaning up temp files in /nsm/elastic-agent-workspace[ERROR   ] Command '/opt/so/__salt.tmp.x465ylqq' failed with return code: 1
[ERROR   ] stdout: 
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 178563, 'retcode': 1, 'stdout': '\nInstallation initiated, view install log for further details.\nNot Accessible\nNot Accessible', 'stderr': ''}
[ERROR   ] Command '/opt/so/__salt.tmp.2i9vu0z4' failed with return code: 1
[ERROR   ] stdout: 
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 179636, 'retcode': 1, 'stdout': '\nInstallation initiated, view install log for further details.\nNot Accessible\nNot Accessible', 'stderr': ''}
      Result: False


[root@so-mngr]# sudo grep ERROR /root/sosetup.log
[ERROR   ] The Salt Master has cached the public key for this node, this salt minion will wait for 10 seconds before attempting to re-authenticate
### Cleaning up temp files in /nsm/elastic-agent-workspace[ERROR   ] Command '/opt/so/__salt.tmp.x465ylqq' failed with return code: 1
[ERROR   ] stdout: 
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 178563, 'retcode': 1, 'stdout': '\nInstallation initiated, view install log for further details.\nNot Accessible\nNot Accessible', 'stderr': ''}
[ERROR   ] Command '/opt/so/__salt.tmp.2i9vu0z4' failed with return code: 1
[ERROR   ] stdout: 
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 179636, 'retcode': 1, 'stdout': '\nInstallation initiated, view install log for further details.\nNot Accessible\nNot Accessible', 'stderr': ''}
[root@so-mngr soc-admin]#

**Comment by jrm360seclab** [comment]
> So you did a fresh OS installation on the same hardware using our ISO image?
> 
> Did you choose the standalone option again?
> 
> Were there any errors during Setup this time?
> 
> Have you checked the logs in /opt/so/log/ for additional clues?
